pkg/apis/keycloak/v1alpha1/keycloakrealm_types.go

```diff
@@ -161,8 +161,16 @@ type TokenResponse struct {
 // KeycloakRealmStatus defines the observed state of KeycloakRealm
 // +k8s:openapi-gen=true
 type KeycloakRealmStatus struct {
-	Phase    string `json:"phase"`
-	LoginURL string `json:"loginUrl"`
+	// Current phase of the operator.
+	Phase StatusPhase `json:"phase"`
+	// Human-readable message indicating details about current operator phase or error.
+	Message string `json:"message"`
+	// True if all resources are in a ready state and all work is done.
+	Ready bool `json:"ready"`
+	// A map of all the secondary resources types and names created for this CR. e.g "Deployment": [ "DeploymentName1", "DeploymentName2" ]
+	SecondaryResources map[string][]string `json:"secondaryResources,omitempty"`
```

Comment:
I guess we don't need this right?

Reply:
I didn't think so at the start either but there is a secret created for each realm user here - https://github.com/keycloak/keycloak-operator/pull/62/files#diff-8ff059c52d814c2fae1d7529c4250040R48.

Reply:
Good point!